armada/handlers/tiller.py

```diff
@@ -156,6 +161,8 @@ def _pre_update_actions(self, actions, namespace):
                     continue
                 LOG.error("Unable to execute name: %s type: %s ", name, type)
         except Exception:
+            raise tiller_exceptions.PreUpdateJobDeleteException(name,
+                                                                namespace)
```

Comment:
@drewwalters96 you should rebase from master and revisit this as this logic has changed significantly.